easybuild/tools/options.py

```diff
@@ -199,8 +201,8 @@ def override_options(self):
             'skip-test-cases': ("Skip running test cases", None, 'store_true', False, 't'),
             'umask': ("umask to use (e.g. '022'); non-user write permissions on install directories are removed",
                       None, 'store', None),
-            'optarch': ("Set architecture optimization, overriding native architecture optimizations",
-                        None, 'store', None),
+            'update-modules-tool-cache': ("Update modules tool cache file(s) after generating module file",
```

Comment:
because now the user cache of the user running EasyBuild is being updated, only to then be totally ignored by EasyBuild itself (since `$LMOD_IGNORE_CACHE` is defined)

this should never have been enabled to start with...